styles/_docs.scss

```
@@ -0,0 +1,352 @@
+.sdk-img {
+  align-self: center;
+}
+
+.light {
+  .sdk-img {
+    content:url(../../img/graphics/sdk-white.png);
+  }
+
+  .ref-book-illustration {
+    content:url(../../img/graphics/ref-book-light.png)
+  }
+
+  .tutorial-illustration {
+    content:url(../../img/graphics/tutorials-illustration-light.png)
+  }
+
+  .concepts-doc-illustration {
+    content:url(../../img/graphics/concepts-docs-light.png)
+  }
+
+  .use-cases {
+    .wallet-illustration {
+      content:url(../../img/graphics/wallet-light.svg)
+    }
+
+    .token-illustration {
+      content:url(../../img/graphics/tokens-light.png)
+    }
+
+    .connections-illustration {
+      content:url(../../img/graphics/nodes-light.svg)
+    }
+  }
+
+  .quickstart-image {
+    content:url(../../img/graphics/getting-started-pages-light.png)
+  }
+
+  .dev-tools-img {
+    content:url(../../img/graphics/dev-tools-light.svg)
+  }
+
+  .dev-tools-link:hover p {
+    color: #000
```

There are a bunch of color name variables defined in `_colors.scss`. Wherever possible, please use those instead of hard-coding color values into the SCSS; that way, if we tweak the color palette, the changes automatically update throughout the styles. (For example, in the past, we changed from "pure black" to "off-black" for some things.)

```suggestion
    color: $black
```